iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -22,67 +22,53 @@
 
     static final String ROOT_ELEMENT_NAME = "shunt";
 
-    private static final String REGULATING_TERMINAL = "regulatingTerminal";
-
     @Override
     protected String getRootElementName() {
         return ROOT_ELEMENT_NAME;
     }
 
     @Override
     protected boolean hasSubElements(ShuntCompensator sc) {
-        return sc != sc.getRegulatingTerminal().getConnectable();
+        return false;
```

Comment:
Is there a bug there (and elsewhere)? If an identifiable has properties, then he has sub elements.

Reply:
No, in `AbstractIdentifiableXml`, we have it checks `if (hasSubElements || identifiable.hasProperty())`, not only what `hasSubElements` returns.